python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/traffic-policies.j2

```diff
@@ -132,16 +132,16 @@ traffic-policies
 {%                         endif %}
 {%                     endfor %}
 {%                 endif %}
-{#                 -- TTL MANAGEMENT -- #}
-{%                 if match.ttl is arista.avd.defined %}
-         ttl {{ match.ttl }}
-{%                 endif %}
 {#                 -- FRAGMENT MANAGEMENT -- #}
 {%                 if match.fragment.offset is arista.avd.defined %}
          fragment offset {{ match.fragment.offset }}
 {%                 elif match.fragment is defined %}
          fragment
 {%                 endif %}
+{#                 -- TTL MANAGEMENT -- #}
```

Comment:
Do check for tcp and udp order

```
match BLUE-C1-POLICY-08 ipv4
         destination prefix 10.0.0.0/8 192.168.0.0/16
         protocol udp destination port 1,10-20
         protocol tcp source port field-set SERVICE-DEMO-SRC destination port field-set SERVICE-DEMO-DST
```

Reply:
Fixed!

Reply:
EOS sorting is not as natural sort hence not adding sorting on protocol